public/data/modules.json

```diff
@@ -58,5 +58,20 @@
     ],
     "link": "https://www.google.com",
     "imageLink": "https://placekitten.com/200/300"
+  },
+  {
+    "id": "39886925-d029-4e2c-a37f-5768f41c2ab4",
+    "makderId": "7b536185-7532-4af9-bcde-4b5595616377",
```

Comment:
This should be makerId, other than that it looks good to go. Tip, from the conversation tab of the pull request you can see what the site will look like. Netlify adds a comment with a site preview link.

Reply:
Fixed! I indeed saw the previous function, but it doesn't appear to work for me. I see a blank page with the right back-ground colour, but no contents. Maybe a browser problem (using chrome right now).

Reply:
I think that was caused by the makerId typo, I'll do some work on making the site a bit more robust to those sorts of typos.